tutorials/euclid_access/Euclid_ERO.md

```diff
@@ -0,0 +1,687 @@
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.7
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Exploring Star Clusters in the Euclid ERO Data
+
+
+## Learning Goals
+By the end of this tutorial, you will be able to:
+
+ &bull; Access the Euclid ERO images using `astroquery`
+
+ &bull; Create cutouts on the large Euclid ERO images directly
+
+ &bull; Extract sources on the Euclid image and run photometry tools
+
+ &bull; Compare photometry to the Gaia catalog
+
+ &bull; Visualize the Euclid ERO image and the Gaia catalog in `Firefly`
+
+
+## Introduction
+Euclid is a European Space Agency (ESA) space mission with NASA participation, to study the geometry and nature of the dark Universe. As part of its first observations, Euclid publicly released so-called *Early Release Observations* (ERO) targeting some press-worthy targets on the sky such as star clusters or local galaxies.
+
+In this notebook, we will focus on the ERO data of **NGC 6397**, a globular cluster 78,000 light years away. (Note that there is another globular cluster, **NGC 6254** that can also be used for this analysis - in fact the user can choose which one to use) The goal of this analysis is to extract the Euclid photometry of the stars belonging to the cluster and compare them to the photometry of Gaia. Due to the different pixel scales of the visible (VIS, $0.1^{\prime\prime}$) and near-IR (NISP, $0.3^{\prime\prime}$) wavelengths, we will first detect and extract the stars in the VIS filter and use their position to subsequently extract the photometry in the NISP (Y, J, H) filters (a method also referred to as *forced photometry*).
+
+One challenge with Euclid data is their size due to the large sky coverage and small pixel size.
+This notebook demonstrates how to download only a cutout of the large Euclid ERO observation image (namely focussing only on the position of the globular cluster. Furthermore, this notebook demonstrates how to extract sources on a large astronomical image and how to measure their photometry across different pixel scales using a very simple implementation of the forced photometry method with positional priors.
+Finally, we also demonstrate how to visualize the Euclid images and catalog in `Firefly`, an open-source web-based UI library for astronomical data archive access and visualization developed at Caltech (https://github.com/Caltech-IPAC/firefly). `Firefly` is a convenient tool to visualize images similar to DS9 on your local computer, but it can run on a cloud-based science platform.
+
+
+This notebook is written to be used in Fornax which is a cloud based computing platform using AWS. The advantage of this is that the user does not need to download actuall data, hence the analysis of large datasets is not limited by local computing resources. It also allows to access data across all archives fast and easy.
```

Comment:
We may want to remove "written to be used in Fornax" because here it's in irsa-tutorials?

also typo in "actually"